.github/CODEOWNERS

```
@@ -1,2 +1,6 @@
 * @MARS-crew/mars
+<<<<<<< work
+*/java @MARS-crew/server-engineer
+=======
 src/main/java @mokhs00 @MARS-crew/server-engineer
+>>>>>>> main
```

앗 이거 force-push 고치다가 충돌난 것 같은데 이렇게 수정 부탁해요 🙏 
```suggestion
* @MARS-crew/mars
src/main/java @mokhs00 @MARS-crew/server-engineer
```